resources/styles/components/questions-dashboard.less

```diff
@@ -23,6 +23,11 @@
     }
   }
 
+  .exercises-display {
+    max-width: @tutor-max-panel-width;
+    margin: 0 auto;
+  }
+
```

Comment:
Ok, it sounds like @kathi-fletcher approved so I'm ok with it.  Just think it looks not-so-great on wider displays and isn't as useful.